### PR TITLE
Feature/applics 80 register to have your say translation 2 of 3

### PR DESCRIPTION
**Pull request by edpackard-work**
## Describe your changes

Translations for the unique 'organisation' pages (and validations) on the register to have your say journey.

https://pins-ds.atlassian.net.mcas.ms/jira/software/c/projects/APPLICS/boards/242?selectedIssue=APPLICS-80

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/forms-web-app/src/locales/en/common.json [hunk 1]
```diff
@@ -75,6 +76,14 @@
...
 			"length": "What you want to tell us must be {{-maxCharacters}} characters or less"
+		},
+		"organisationName": {
+			"empty": "Enter your organisation or charity name",
+			"length": "Name of your organisation or charity name must be 255 characters or less"
+		},
+		"jobTitleOrRole": {
+			"empty": "Enter your job title or volunteer role",
+			"length": "Your job title or volunteer role must be 64 characters or less"
 		}
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/locales/en/common.json:87]: Would these translations be better placed in a common register folder scoped to the journey?

File: packages/forms-web-app/src/pages/projects/register/_translations/translations.test.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+describe('pages/cookies/_translations', () => {
+	it('should return the correct English translations', () => {
+		expect(enCookiesTranslations).toMatchSnapshot();
+	});
+
+	it('should return the correct Welsh translations', () => {
+		expect(cyCookiesTranslations).toMatchSnapshot();
+	});
+
+	it('should return the same keys for English and Welsh translations', () => {
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/pages/projects/register/_translations/translations.test.js:13]: Is this test passing? I have noticed issues where this on my local main where this test is not failing the coverage test but isn't really passing. I think it should look like this:

`it('should returning matching keys for both English and Welsh translations', () => {
    expect(Object.keys(enTranslations).sort()).toStrictEqual(Object.keys(cyTranslations).sort());
});`

Comment (edpackard-work) [reply packages/forms-web-app/src/pages/projects/register/_translations/translations.test.js:13]: resolved per discussion - the snapshot testing / testing of nested keys will be addressed in a future ticket

File: packages/forms-web-app/src/locales/en/common.json (hunk 1 above)
Comment (edpackard-work) [reply packages/forms-web-app/src/locales/en/common.json:87]: good idea - I'll move them

**Review by christopher-semmens — APPROVED** [review]

**Pull request merged**